src/components/SelectionList/BaseSelectionList.js

```diff
@@ -134,15 +134,11 @@ function BaseSelectionList({
     }, [canSelectMultiple, sections]);
 
     const [focusedIndex, setFocusedIndex] = useState(() => {
-        const defaultIndex = 0;
-
         const indexOfInitiallyFocusedOption = _.findIndex(flattenedSections.allOptions, (option) => option.keyForList === initiallyFocusedOptionKey);
 
         if (indexOfInitiallyFocusedOption >= 0) {
             return indexOfInitiallyFocusedOption;
         }
-
-        return defaultIndex;
```

Comment:
isn't this function missing a return at the end? `return undefined;`? did you want to return `undefined`?

Reply:
I actually had that before, but we don't need to explicitly `return undefined` because if you access the return value of a function that doesn't return anything it'll be `undefined`

Do you think it's better to add back that `return undefined`?

Reply:
My personal preference is to have explicit `return undefined;`, I find it weird when there is nothing and you have to know this "detail" of javascript to know what is happening.

Having said that, I don't know what the guidelines say about this.

Reply:
No need to define the variable `defaultIndex` though